python/mxnet/gluon/block.py

```diff
@@ -1635,6 +1637,47 @@ def reset_ctx(self, ctx):
         for p in params.values():
             p.reset_ctx(ctx)
 
+    def mark_vars(self, var_arrays):
+        """Mark the intermediate nodes
+
+        Parameters
+        ----------
+        vars: NDArrays or List[NDArrays] with nonempty deferredcomputation_entry_
```

Comment:
Could you follow the [example](https://github.com/apache/incubator-mxnet/blob/49c47380fac3c8c8e7ee8eb237000165e22828ef/python/mxnet/numpy/multiarray.py#L2621-L2645) to write the docstring for mark_vars?

Reply:
Done.